src/main/java/seedu/duke/command/MemberCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.duke.command;
+
+import java.util.ArrayList;
+
+public class MemberCommand {
+    public void addMemberCommand(ArrayList<String> names) {
+        /*
```

Comment:
SO do we insert the code for adding and editting to project member list here?

Reply:
There will be another class that runs the code so that we can lessen the code in Project class

Reply:
Yes so something like project member which will be handling all this?

Reply:
Good job, as @chocomango  said we need to break commands into subclasses I think.

Reply:
I agree, we should split them into subclasses, inheriting `MemberCommand` for example. It's what the TA suggested.